common/djangoapps/student/management/commands/bulk_unenroll.py

```diff
@@ -22,20 +22,31 @@ class Command(BaseCommand):
     It expect that the data will be provided in a csv file format with
     first row being the header and columns will be either one of the
     following:
-    username,course-id
+    | username | course_id |
     OR
-    course-id
+    | course_id |
+
+    Example:
+            $ ... bulk_unenroll --csv_path=foo.csv
+            $ ... bulk_unenroll --csv_path=foo.csv --commit
     """
+    commit = False
 
     def add_arguments(self, parser):
-        parser.add_argument('-p', '--csv_path',
-                            metavar='csv_path',
-                            dest='csv_path',
-                            required=False,
-                            help='Path to CSV file.')
+        parser.add_argument(
+            '-p', '--csv_path',
+            metavar='csv_path',
+            dest='csv_path',
+            required=True,
+            help='Path to CSV file.')
+        parser.add_argument(
+            '--commit',
+            action='store_true',
+            help='Save the changes, without this flag only a dry run will be performed and nothing will be changed')
 
     def handle(self, *args, **options):
         csv_path = options['csv_path']
+        self.commit = options['commit']
```

Comment:
commit required is not set, so this will fail if commit is not provided. Use `get` to obtain commit value.

Reply:
```
> python manage.py lms bulk_unenroll --csv_path=awais.csv --commit
> options
{'pythonpath': None, 'csv_path': 'awais.csv', 'no_color': False, 'force_color': False, 'traceback': False, 'commit': True, 'settings': None, 'verbosity': 1}
```

and without commit.
```
> python manage.py lms bulk_unenroll --csv_path=awais.csv
> options
{'pythonpath': None, 'csv_path': 'awais.csv', 'no_color': False, 'force_color': False, 'traceback': False, 'commit': False, 'settings': None, 'verbosity': 1}
```